frb_dart/lib/src/basic.dart

```diff
@@ -48,10 +48,7 @@ abstract class FlutterRustBridgeBase<T extends FlutterRustBridgeWireBase> {
 
   void _sanityCheckSingleton() {
     if (_instances.contains(runtimeType)) {
-      throw Exception(
-        'Subclasses of `FlutterRustBridgeBase` should be singletons - '
-        'there should not be two instances (runtimeType=$runtimeType)',
-      );
+      return;
```

Comment:
:eyes: what is happening here, it seems we should ensure it is singleton

Reply:
The brief reason is stated here:https://github.com/fzyzcjy/flutter_rust_bridge/pull/1145#issuecomment-1504915799.